.github/workflows/main.yml

```diff
@@ -16,7 +16,7 @@ jobs:
   test_basis:
     runs-on: ubuntu-latest
     env:
-      OPTIONS: "COV=false LOG_LEVEL=info SKIP_DICHO=true SKIP_REAL_CASES=true SKIP_PERIODIC=true SKIP_SPLIT_CLUSTERING=true"
+      OPTIONS: "CI=true COV=false LOG_LEVEL=info SKIP_DICHO=true SKIP_REAL_CASES=true SKIP_PERIODIC=true SKIP_SPLIT_CLUSTERING=true"
```

Comment:
This is not necessary, the environment variable *CI* is always true in Github Action : https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables Sorry if I wasn't clear. You can remove for all jobs.